Main.py

```diff
@@ -104,6 +104,10 @@ def resolve_settings(settings, window=dummy_window()):
             raise Exception(f'Player Num is {settings.player_num}; must be between (1, {settings.world_count})')
         settings.player_num = settings.world_count
 
+    # Set to a custom hint distribution if plando is overriding the distro
+    if len(settings.hint_dist_user) == 0:
+        settings.hint_dist = 'custom'
```

Comment:
This is apparently causing a crash on my end; I'm guessing because you're doing this in `resolve_settings` which is before `generate`, when we build the World, it looks for a "custom.json" hint dist file, which I don't have. Not sure how to correctly handle it, unless you have it backwards and it should be `!= 0`?

Reply:
Yep, copy/paste error from the World constructor. I don't know why this worked on my side.